README.md

```diff
@@ -2,10 +2,13 @@
   <img src="./docs/Logo.png" width="300px"/>
   <br />
   <br />
+  <img src="https://img.shields.io/badge/Author-%40americanairlines-blue" />
+  <img src="https://img.shields.io/badge/Version-v3-%5B0%2C0%2C255%5D" />
+  <img src="../../workflows/Build/badge.svg" />
```

Comment:
Just FYI, this is gonna pull from the default branch, so technically this will be incorrect until we change the default branch. I'm ok with that, just wanted you to be aware.

Reply:
Oh ok that's an issue then... can you please modify the path here to target THAT build and then also change the pattern matching in the build action to trigger on v3 commits as well?

Reply:
Oh jk I think I was being silly... workflow should have been "build" instead of "Build"